### DISCUSSION TITLE
d2-ui and d2-ui-rich-text migration status

### DISCUSSION THREAD

**Discussion by simonadomnisoru** [dhis2/notes#334] [Q&A] [answered]
Hey,

I am not able to find in the [@dhis2/ui Storybook](https://ui.dhis2.nu/demo/?path=/story/) examples for the [d2-ui-rich-text](https://github.com/dhis2/d2-ui/tree/master/packages/rich-text) old components. Do you know if these components were migrated or if we still have to rely on the deprecated [d2-ui](https://github.com/dhis2/d2-ui) repository?

Thank you!

**Comment by ghost** [comment, marked as answer]
I don't think we have a rich text component in ui at the moment. The issue is still open here: https://dhis2.atlassian.net/browse/TECH-294
